server/routes/bookedtoday/unexpectedArrivals/addPersonalDetailsController.ts

```diff
@@ -0,0 +1,35 @@
+import type { RequestHandler, Response, Request } from 'express'
+import { convertToTitleCase } from '../../../utils/utils'
+import { State } from '../arrivals/state'
+
+export default class AddPersonalDetailsController {
+  public view(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      res.render('pages/unexpectedArrivals/addPersonalDetails.njk', {
+        errors: req.flash('errors'),
+        data: req.flash('input')[0],
+      })
+    }
+  }
+
+  public submit(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      if (req.errors) {
+        req.flash('input', req.body)
+        return res.redirect('/manually-confirm-arrival/add-personal-details')
+      }
+
+      const { firstName, lastName, year, month, day, sex } = req.body
+      const dateOfBirth = `${year.padStart(4, '0')}-${month.padStart(2, '0')}-${day.padStart(2, '0')}`
+
+      State.newArrival.set(res, {
+        firstName: convertToTitleCase(firstName),
+        lastName: convertToTitleCase(lastName),
+        dateOfBirth,
+        sex,
+      })
+
+      return res.redirect('/prisoners/unexpected-arrivals/imprisonment-status')
```

Comment:
the 'unexpected-arrivals' segment in the url is temporary until we decide what will be used

Reply:
this should go to sex controller as thats the beginning of the pipeline that all the flows go to.

It will just redirect through as you've provided the sex but we're considering it almost as a separate app and thats the entry point

Reply:
yes. changed